### DISCUSSION TITLE
Add button for adding (sub)folders to asset volumes

### DISCUSSION THREAD

**Discussion by rensverschuren** [craftcms/cms#8438] [Ideas]
A lot of our clients (even some of my colleagues) don’t know that they have to right click on a asset folder to create a subfolder in it. Maybe it would be great if the assets section in the control panel gets a button to add subfolders. 
Primary UI actions shouldn’t be accessible using right click only, in my opinion.

**Comment by brandonkelly** [comment]
> Primary UI actions shouldn’t be accessible using right click only, in my opinion.

Agree
